web_select_all_companies/static/src/xml/switch_all_company_menu.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t
+        t-name="web_select_all_companies.SwitchAllCompanyMenu"
+        t-inherit="web.SwitchCompanyMenu"
+        t-inherit-mode="extension"
+        owl="1"
+    >
+        <xpath expr="//t[@t-key='company.id']" position="before">
+            <DropdownItem class="'p-0'">
+                <div class="d-flex">
+                    <div
+                        role="menuitemcheckbox"
+                        tabindex="0"
+                        t-att-class="{ 'border-primary': isCurrent }"
+                        class="border-end toggle_company"
+                        t-on-click.stop="() => this.toggleSelectAllCompanies()"
+                    >
+                            <span class="btn border-0 p-2">
+                                <i
+                                class="fa fa-fw py-2"
+                                t-att-class="isAllCompaniesSelected ? 'fa-check-square text-primary' : 'fa-square-o'"
+                            />
+                            </span>
+                    </div>
+                    <div
+                        role="button"
+                        tabindex="0"
+                        class="d-flex flex-grow-1 align-items-center py-0 log_into ps-2"
```

Comment:
```suggestion
                        class="d-flex flex-grow-1 align-items-center py-0 ps-2"
```

IMHO `log_into`has to be removed here, since it is (was?) linked to JS event to "log into a company" and it is irrelevant for "All companies".